src/github/api/client.rs

```diff
@@ -13,6 +14,7 @@ use crate::github::{Branch, CommitSha, GithubRepoName, PullRequest, PullRequestN
 
 /// Provides access to a single app installation (repository) using the GitHub API.
 pub struct GithubRepositoryClient {
+    pub app: App,
```

Comment:
I tried to save just app_html_url, but it does not make sense, so I saved the entire App instead

Reply:
I wouldn't mind caching just the URL, but storing the App is also fine. Thanks!